With @deprecated before params gen_wrappers fails to read return type and its attributes
--- liblinphone-5.3.32/include/linphone/call_params.h.orig	2024-03-09 13:44:49.000000000 +0100
+++ liblinphone-5.3.32/include/linphone/call_params.h	2024-03-11 20:21:53.429493890 +0100
@@ -60,9 +60,9 @@ LINPHONE_PUBLIC const char *linphone_cal
  * Copy an existing #LinphoneCallParams object to a new #LinphoneCallParams object.
  * linphone_call_params_copy() is error-prone, leading to inconsistent parameters being passed to
  * linphone_core_invite_address_with_params() or linphone_call_accept_with_params().
- * @deprecated use exclusively linphone_core_create_call_params() to create #LinphoneCallParams object.
  * @param call_params The #LinphoneCallParams object to copy. @notnil
  * @return A copy of the #LinphoneCallParams object. @notnil @tobefreed
+ * @deprecated use exclusively linphone_core_create_call_params() to create #LinphoneCallParams object.
  **/
 LINPHONE_PUBLIC LINPHONE_DEPRECATED LinphoneCallParams *
 linphone_call_params_copy(const LinphoneCallParams *call_params);
